Microwave.Classes/Boundary/Buzzer.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microwave.Classes.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microwave.Classes.Boundary;
+
+public class buzzer : IBuzzer
+{
+
+    public void CookingIsEndedSound()
+    {
+        Console.WriteLine("Ding, Ding, Ding! Cooking Done\a");
```

Comment:
I stedet for Console.WriteLine skal vi så ikke bruge IOutput klassen med Outputline metoden. Det bliver også gjort i Light klassen.